<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>622</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>随便聊</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image_resouce.qrc">
    <normaloff>:/3440/pp.jpg</normaloff>:/3440/pp.jpg</iconset>
  </property>
  <property name="windowOpacity">
   <double>2.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QPalette{background:#F0F0F0;}

QWidget#frmMain,QWidget[Form=&quot;true&quot;]{
border:1px solid #575757;
border-radius:0px;
}

.QFrame{
border:1px solid #C2CCD8;
border-radius:5px;
}

QWidget#widget_title,QToolBar{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #4D4D4D,stop:1 #292929);
}

QLabel{
color:#000000;
}

QLabel#lab_Ico,QLabel#lab_Title{
border-radius:0px;
color:#F0F0F0;
background-color:rgba(0,0,0,0);
border-style:none;
}

QLabel#labfrmData,QLabel#labfrmHost,QLabel#labfrmDefence,QLabel#labfrmIPCGroup,QLabel#labfrmIPC,QLabel#labfrmIPCLink,QLabel#labfrmVideoPlayBack,QLabel#labfrmConfig,QLabel#labCheckAllStatus,QLabel#labfrmExit{
padding:5px;
color:#F0F0F0;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #4D4D4D,stop:1 #292929);
}

QLabel#labfrmData:hover,QLabel#labfrmHost:hover,QLabel#labfrmDefence:hover,QLabel#labfrmIPCGroup:hover,QLabel#labfrmIPC:hover,QLabel#labfrmIPCLink:hover,QLabel#labfrmVideoPlayBack:hover,QLabel#labfrmConfig:hover,QLabel#labCheckAllStatus:hover,QLabel#labfrmExit:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #636363,stop:1 #575757);
}

QLabel[labVideo=&quot;true&quot;]{
color:#F0F0F0;
border:1px solid #575757;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #4D4D4D,stop:1 #292929);
}

QLabel[labVideo=&quot;true&quot;]:focus{
border:1px solid #FF0000;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #636363,stop:1 #575757);
}

QLineEdit,QTextEdit,QSpinBox{
border:1px solid #C2CCD8;
border-radius:5px;
padding:2px;
background:none;
selection-background-color:#4D4D4D;
selection-color:#F0F0F0;
}

QLineEdit[echoMode=&quot;2&quot;]{
lineedit-password-character:9679;
}

.QGroupBox{
border:1px solid #C2CCD8;
border-radius:5px;
}

.QToolButton{
border-style:none;
color:#F0F0F0;
padding:5px;
min-height:15px;
min-width:30px;
border-radius:5px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #4D4D4D,stop:1 #292929);
}

.QToolButton:hover{
border:1px solid #C2CCD8;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #636363,stop:1 #575757);
}

.QPushButton{
border-style:none;
border:1px solid #C2CCD8;
color:#F0F0F0;
padding:5px;
min-height:20px;
border-radius:5px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #4D4D4D,stop:1 #292929);
}

.QPushButton[focusPolicy=&quot;0&quot;]{
border-style:none;
border:1px solid #C2CCD8;
color:#F0F0F0;
padding:0px;
min-height:10px;
border-radius:3px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #4D4D4D,stop:1 #292929);
}

.QPushButton:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #636363,stop:1 #575757);
}

.QPushButton:pressed{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #4D4D4D,stop:1 #292929);
}

.QPushButton:disabled{
color:#838383;
background:#F4F4F4;
}

QPushButton#btnMenu,QPushButton#btnMenu_Min,QPushButton#btnMenu_Max,QPushButton#btnMenu_Close{
border-radius:0px;
color:#F0F0F0;
background-color:rgba(0,0,0,0);
border-style:none;
}

QPushButton#btnMenu:hover,QPushButton#btnMenu_Min:hover,QPushButton#btnMenu_Max:hover{
background-color:qlineargradient(spread:pad,x1:0,y1:1,x2:0,y2:0,stop:0 rgba(25,134,199,0),stop:1 #636363);
}

QPushButton#btnMenu_Close:hover{
background-color:qlineargradient(spread:pad,x1:0,y1:1,x2:0,y2:0,stop:0 rgba(238,0,0,128),stop:1 rgba(238,44,44,255));
}

QCheckBox{
color:#000000;
spacing:2px;
}

QCheckBox::indicator{
width:20px;
height:20px;
}

QCheckBox::indicator:unchecked{
image:url(:/image/checkbox_unchecked.png);
}

QCheckBox::indicator:checked{
image:url(:/image/checkbox_checked.png);
}

QRadioButton{
color:#000000;
spacing:2px;
}

QRadioButton::indicator{
width:15px;
height:15px;
}

QRadioButton::indicator::unchecked{
image:url(:/image/radio_normal.png);
}

QRadioButton::indicator::checked{
image:url(:/image/radio_selected.png);
}

QSpinBox::up-button{
image:url(:/image/array_up.png);
}

QSpinBox::down-button{
image:url(:/image/array_down.png);
}

QComboBox,QDateEdit,QSpinBox{
border-radius:3px;
padding:3px 5px 3px 5px;
border:1px solid #C2CCD8;
}

QComboBox::drop-down,QDateEdit::drop-down{
subcontrol-origin:padding;
subcontrol-position:top right;
width:15px;
border-left-width:1px;
border-left-style:solid;
border-top-right-radius:3px;
border-bottom-right-radius:3px;
border-left-color:#C2CCD8;
}

QComboBox::down-arrow,QDateEdit::down-arrow{
image:url(:/image/array_down.png);
}

QMenu{
color:#F0F0F0;
background-color:#4D4D4D;
margin:2px;
}

QMenu::item{
padding:3px 10px 3px 10px;
}

QMenu::indicator{
width:13px;
height:13px;
}

QMenu::item:selected{
color:#F0F0F0;
border:1px solid #575757;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #4D4D4D,stop:1 #292929);
}

QMenu::separator{
height:1px;
background:#575757;
}

QProgressBar{
background:#C2CCD8;
border-radius:5px;
text-align:center;
border:1px solid #C2CCD8;
}

QProgressBar::chunk{
width:5px;
margin:0.5px;
background-color:#4D4D4D;
}

QSlider::groove:horizontal,QSlider::add-page:horizontal{
height:8px;
border-radius:3px;
background:#C2CCD8;
}

QSlider::sub-page:horizontal{
height:8px;
border-radius:3px;
background:#575757;
}

QSlider::handle:horizontal{
width:13px;
margin-top:-3px;
margin-bottom:-3px;
border-radius:6px;
background:qradialgradient(spread:pad,cx:0.5,cy:0.5,radius:0.5,fx:0.5,fy:0.5,stop:0.6 #FFFFFF,stop:0.8 #575757);
}

QSlider::groove:vertical,QSlider::sub-page:vertical{
width:8px;
border-radius:3px;
background:#C2CCD8;
}

QSlider::add-page:vertical{
width:8px;
border-radius:3px;
background:#575757;
}

QSlider::handle:vertical{
height:13px;
margin-left:-2px;
margin-right:-3px;
border-radius:6px;
background:qradialgradient(spread:pad,cx:0.5,cy:0.5,radius:0.5,fx:0.5,fy:0.5,stop:0.6 #FFFFFF,stop:0.8 #575757);
}

QScrollBar:vertical{
width:10px;
background-color:rgba(0,0,0,0%);
padding-top:10px;
padding-bottom:10px;
}

QScrollBar:horizontal{
height:10px;
background-color:rgba(0,0,0,0%);
padding-left:10px;
padding-right:10px;
}

QScrollBar::handle:vertical,QScrollBar::handle:horizontal{
width:10px;
background:#575757;
}

QScrollBar::handle:vertical:hover,QScrollBar::handle:horizontal:hover{
width:10px;
background:#292929;
}

QScrollBar::add-line:vertical{
height:10px;
width:10px;
subcontrol-position:bottom;
subcontrol-origin:margin;
border-image:url(:/image/add-line_vertical.png);
}

QScrollBar::add-line:horizontal{
height:10px;
width:10px;
subcontrol-position:right;
subcontrol-origin:margin;
border-image:url(:/image/add-line_horizontal.png);
}

QScrollBar::sub-line:vertical{
height:10px;
width:10px;
subcontrol-position:top;
subcontrol-origin:margin;
border-image:url(:/image/sub-line_vertical.png);
}

QScrollBar::sub-line:horizontal{
height:10px;
width:10px;
subcontrol-position:left;
subcontrol-origin:margin;
border-image:url(:/image/sub-line_horizontal.png);
}

QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical,QScrollBar::add-page:horizontal,QScrollBar::sub-page:horizontal{
width:10px;
background:#C2CCD8;
}

QScrollArea{
border:0px;
}

QTreeView,QListView,QTableView,QTabWidget::pane{
border:1px solid #C2CCD8;
selection-background-color:#636363;
selection-color:#F0F0F0;
}

QTableView::item:selected,QListView::item:selected,QTreeView::item:selected{
color:#F0F0F0;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #4D4D4D,stop:1 #292929);
}

QTableView::item:hover,QListView::item:hover,QTreeView::item:hover{
color:#F0F0F0;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #636363,stop:1 #575757);
}

QTableView::item,QListView::item,QTreeView::item{
padding:5px;
margin:0px;
}

QHeaderView::section,QTableCornerButton:section{
padding:3px;
margin:0px;
color:#F0F0F0;
border:1px solid #575757;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #636363,stop:1 #575757);
}

QTabBar::tab{
border-radius:5px;
border:1px solid #C2CCD8;
color:#F0F0F0;
min-width:60px;
min-height:20px;
padding:3px 8px 3px 8px;
margin:1px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #636363,stop:1 #575757);
}

QTabBar::tab:selected,QTabBar::tab:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #4D4D4D,stop:1 #292929);
}

QStatusBar::item{
border:0px solid #4D4D4D;
border-radius:3px;
}

QToolBox::tab{
padding:5px;
border-radius: 5px;
color:#F0F0F0;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #4D4D4D,stop:1 #292929);
}

QToolBox::tab:selected{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #636363,stop:1 #575757);
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>560</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>12</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtName">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>163</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入你的昵称</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtContext">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>522</y>
     <width>832</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>说点声明把</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtShow">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>581</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="tabStopWidth">
    <number>80</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pic00">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>表情</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pic">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>呆呆着看着你</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改昵称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPerson">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入私聊对象昵称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>修改昵称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>80</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>找人私聊：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>私发</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询在线</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>120</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>在线统计：</string>
   </property>
  </widget>
  <widget class="QLabel" name="chatSelect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(224, 255, 226);
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>群聊模式</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>退出房间：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitChat">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出房间</string>
   </property>
  </widget>
  <widget class="QComboBox" name="chatLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string>功能</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>本地记录</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>关于我</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>250</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查看链表结构</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image_resouce.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>sendMesage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sendMesage()</slot>
 </slots>
</ui>
